### DISCUSSION TITLE
v1.7.5

### DISCUSSION THREAD

**Discussion by jdevcs** [ethereum/web3.js#5302] [Announcements]
### Changed
-  Replace xhr2-cookies deps to cross-fetch for web3-providers-http (#5085)

### Added
-  Documentation details about `maxFeePerGas` and `maxPriorityFeePerGas` (#5121)
-  Added `createAccessList` types in web3.eth (#5146)

### Fixed
-  Improving `AbstractProvider` interface (#5150)
-  Fix typos in web3-eth-accounts.rst & TESTING.md (#5047)
-  Fix remove wallet using an index when an account address and address lowercase are equal (#5049)
-  Improve README.md & Fix typos (#4848)
-  Add optional hex formatting parameter for getTransactionrReceipt (#5153)
-  Fix transactionRoot -> transactionsRoot in BlockHeader (#5083)
-  Fix Promise in Accounts.signTransaction() throwing errors that cannot be caught (#4724)
-  Fixed unit tests & removed dead code for web3-providers-http (#5228)

### Security
-  Updated `got` lib version and fixed other libs using npm audit fix (#5178) (#5254)

<hr /><em>This discussion was created from the release <a href='https://github.com/ChainSafe/web3.js/releases/tag/v1.7.5'>v1.7.5</a>.</em>

**Comment by mconnelly8** [comment]
Thank you to all of the first time community contributors to the Web3.js project!  

[@cyborgRen](https://github.com/cyborgRen) made their first contribution in [#5145](https://github.com/ChainSafe/web3.js/pull/5145)
[@sangwon0001](https://github.com/sangwon0001) made their first contribution in [#5142](https://github.com/ChainSafe/web3.js/pull/5142)
[@superMDguy](https://github.com/superMDguy) made their first contribution in [#5116](https://github.com/ChainSafe/web3.js/pull/5116)
[@Neurone](https://github.com/Neurone) made their first contribution in [#5121](https://github.com/ChainSafe/web3.js/pull/5121)
[@MujeebullahKalwar](https://github.com/MujeebullahKalwar) made their first contribution in [#5115](https://github.com/ChainSafe/web3.js/pull/5115)

We appreciate your contributions!
